OCExtensions/Classes/Image/GeneralImageExtension.swift

```diff
@@ -30,4 +30,27 @@ public extension UIImage {
         }
     }
     
+    func inset(by inset: CGFloat) -> UIImage {
```

Comment:
You can change the name of the function from `inset` to `insets` since it is setting all insets of the View.